src/components/course/repo/helper.js

```diff
@@ -0,0 +1,15 @@
+// todo test?
+
+const updateManyResult = ({ n, nModified, ok }) => {
+	// todo is ok === 0 possible or does it throw?
+	if (ok !== 1) {
+		// log/throw
+	}
+	return { success: ok === 1, matchedDocuments: n, modifiedDocuments: nModified };
```

Comment:
should not return matchedDocuments, remove it
should produce warn log ok !== 1 or throw